mne_g3d/viz.py

```diff
@@ -4,8 +4,8 @@
 import numpy as np
 
 
-def read_brain_mesh(surface_path: str):
-    """Function reads triangular format Freesurfer brain surface.
+def read_brain_mesh(surface_path):
+    u"""Read triangular format Freesurfer brain surface.
 
```

Comment:
I don't understand the purpose of the `u"""`, why is it different from regular multiple line strings?  Does it mean unicode?

Reply:
Yes it makes explicit that there is `unicode` in there (usually we try to avoid it if possible, but it's fine for now)

Reply:
Hm, I don't understand why is it required to be explicit for Python 3, since all the string literals are Unicode,  unless otherwise specified. Is it for downgrading code base to Python 2 or something similar?

Reply:
It is not required for 3 only 2. The style checker ensures code is compatible for both.

Reply:
ok